tests/test_engine.py

```diff
@@ -419,11 +419,11 @@ async def test_duplicate_rule_names():
 
 @pytest.mark.asyncio
 async def test_empty_rule_names():
-    with pytest.raises(RulenameEmptyException):
+    with pytest.raises(ValidationError):
```

Comment:
I have detected a bug in this test. The rulebook file doesn't contain a value for "name" field, so yaml parser returns None instead of an empty string. It must have a `""` to force the empty string. Then, I think the test will fail, because the schema would accept an empty string, the schema should have defined the `name` field with min_length attr.

Reply:
@Alex-Izquierdo I have added a minLength and pattern to the rule name.